integration-libs/cdc/core/store/effects/cdc-user-addresses.effect.spec.ts

```diff
@@ -298,26 +294,20 @@ describe('CDC User Addresses effect', () => {
         status: 'ERROR',
         errorMessage: 'Error deleting default address in CDC',
       };
-      const expected = cold('-b', {
-        b: error,
-      });
+      const expected = cold('-#', null, error);
 
       spyOn(cdcJSService, 'updateAddressWithoutScreenSet').and.returnValue(
-        of(error)
+        throwError(error)
       );
 
-      cdcUserAddressesEffect.cdcDeleteUserAddress$.subscribe(() => {
-        expect(cdcJSService.updateAddressWithoutScreenSet).toHaveBeenCalled();
-        cdcJSService
-          .updateAddressWithoutScreenSet(mockUserAddress.formattedAddress + '')
-          .subscribe({
-            error: (error) => {
-              expect(globalMessageService.add).toHaveBeenCalledWith(
-                { key: error.errorMessage },
-                GlobalMessageType.MSG_TYPE_ERROR
-              );
-            },
-          });
+      cdcUserAddressesEffect.cdcDeleteUserAddress$.subscribe({
+        error: (error) => {
+          expect(cdcJSService.updateAddressWithoutScreenSet).toHaveBeenCalled();
+          expect(globalMessageService.add).toHaveBeenCalledWith(
+            error.errorMessage,
+            GlobalMessageType.MSG_TYPE_ERROR
+          );
```

Comment:
Please note that those expects are not executed at all!
Proof: if you add `done()` callback here, you will get timeout error:
<img width="1346" alt="image" src="https://user-images.githubusercontent.com/4001059/229102644-6c5463c6-0397-4870-b9ea-723f9992aff1.png">

I explained why it happens and I suggested an alternative approach in this example PR: https://github.com/SAP/spartacus/pull/17049

PS. Similar remark applies to other tests in this file.

Reply:
Hi @Platonn,

The PR: [17049](https://github.com/SAP/spartacus/pull/17049) is already merged to develop.
We can still do the marble testing, because if the effects change in the future in the core, these tests will help us catch it.

Let me try to fix the marbles test(s) by removing the subscriptions..

thanks.